templates/expo-template-bare-minimum/ios/Podfile

```diff
@@ -6,6 +6,12 @@ podfile_properties = JSON.parse(File.read(File.join(__dir__, 'Podfile.properties
 
 ENV['RCT_NEW_ARCH_ENABLED'] = podfile_properties['newArchEnabled'] == 'true' ? '1' : '0'
 ENV['EX_DEV_CLIENT_NETWORK_INSPECTOR'] = podfile_properties['EX_DEV_CLIENT_NETWORK_INSPECTOR']
+if ENV['EX_UPDATES_NATIVE_DEBUG'] != '1'
+  ENV['EX_UPDATES_NATIVE_DEBUG'] = podfile_properties['updatesNativeDebug'] == 'true' ? '1' : '0'
+end
+if ENV['EX_UPDATES_CUSTOM_INIT'] != '1'
+  ENV['EX_UPDATES_CUSTOM_INIT'] = podfile_properties['updatesCustomInit'] == 'true' ? '1' : '0'
+end
```

Comment:
a podspec can already read podfile_properties, so we don't have to pull this into template
https://github.com/expo/expo/blob/68db114f9d0da492f328a0d0681fe24231466d2c/packages/expo-updates/ios/EXUpdates.podspec#L4

Reply:
Fixed.